Lib/test/test_ensurepip.py

```diff
@@ -60,6 +60,11 @@ def setUp(self):
         self.run_pip.return_value = 0
         self.addCleanup(run_pip_patch.stop)
 
+        # Ensure zlib is available for tests (unless specifically testing missing zlib)
```

Comment:
```suggestion
        # Allow testing on zlib-less platforms by avoiding the check for zlib in _bootstrap()
```
Paul, would you find this clearer?